beacon-chain/rpc/prysm/v1alpha1/validator/BUILD.bazel

```diff
@@ -173,18 +176,86 @@ go_test(
         "proposer_attestations_test.go",
         "proposer_bellatrix_test.go",
         "proposer_deposits_test.go",
+        "proposer_empty_block_test.go",
         "proposer_execution_payload_test.go",
+        "proposer_exists_test.go",
+        "proposer_slashings_test.go",
         "proposer_sync_aggregate_test.go",
         "proposer_test.go",
+        "server_mainnet_test.go",
         "server_test.go",
+        "status_mainnet_test.go",
         "status_test.go",
         "sync_committee_test.go",
         "validator_test.go",
     ],
     embed = [":go_default_library"],
     eth_network = "minimal",
     tags = ["minimal"],
-    deps = common_deps,
```

Comment:
Why change this instead of adding to common_deps?

Reply:
No reason at all. My original workflow broke it. 
- uncomment ignore
- gazella
- revert unwanted changes (I missed this here)
- comment ignore back

I'll revert this now. Sorry